client/src/components/App.jsx

```diff
@@ -18,7 +18,7 @@ class App extends React.Component {
   
   componentDidMount() {
     const component = this; 
-    axios('http://localhost:3000/locations/1/reviews')
+    axios('http://localhost:3000/locations/2/reviews')
```

Comment:
What is the plan to make this dynamic?

Reply:
@Pfdiamond13 I'm still uncertain, will be sure to consult with someone on this.